ci/builders/README.md

```diff
@@ -154,231 +156,252 @@ An archive component is used to tell the recipes which artifacts are generated b
-
-```
-          "tests": [
-            {
-                "language": "python3",
-                "name": "Host Tests for host_debug_impeller_vulkan",
-                "parameters": [
-                    "--variant",
-                    "host_debug_impeller_vulkan",
-                    "--type",
-                    "impeller-vulkan",
-                    "--engine-capture-core-dump"
-                ],
-                "script": "flutter/testing/run_tests.py",
-                "type": "local"
-            }
-        ]
+In the example above the ninja command will use the configuration for
+host\_debug and will build artifacts and embedder targets as described
+by the
+[flutter/build/archives/BUILD.gn](https://github.com/flutter/engine/blob/main/build/archives/BUILD.gn)
+file.
+
+#### Tests
+
+This section of the build configuration is also known as local tests. It
+contains a list of dictionaries with configurations for scripts and
+parameters used to run tests inside the current build unit. These tests
+should not reference or use anything outside of the commit checkout or
+the outputs generated by running gn and ninja sections of the build
+configuration.
+
+```yaml
+"tests": [
+   {
+       "language": "python3",
+       "name": "Host Tests for host_debug_impeller_vulkan",
+       "parameters": [
+           "--variant",
+           "host_debug_impeller_vulkan",
+           "--type",
+           "impeller-vulkan",
+           "--engine-capture-core-dump"
+       ],
+       "script": "flutter/testing/run_tests.py"
+   }
+]
 ```
 
 Description of the fields:
 
-* **Language,** the executable used to run the script, e.g. python3.
-* **Name,** the name of the step running the script.
-* **Parameters**, flags or parameters passed to the script.
-* **Script**, the path to the script to execute relative to the checkout directory.
-* **Type**, the test type. (Deprecate? Test on the build config will be always local)
-
-The test scripts will run in a deferred context (failing the step only after logs have been uploaded). The tester and builder recipes provide an environment variable called FLUTTER\_LOGS\_DIR pointing a temporary directory where the test runner can place any logs|artifacts needed to debug issues. At the end of the test execution the content of FLUTTER\_LOGS\_DIR will be uploaded to Google Cloud Storage before signaling the pass | fail test state.
-
-Note that to keep the recipes generic they don’t know anything about what the test script is doing and it is the responsibility of the test script to copy the relevant files to the FLUTTER\_LOGS\_DIR directory.
+* **language** - the executable used to run the script, e.g. python3, bash.
+In general any executable found in the path can be used as language. The
+default value for this property is `bash`.
```

Comment:
Taking that comment back, default is empty which means script will be treated as executable with the assumption that already has the right permissions.

Bash can be used on all the platforms the caveat is the use of paths in the windows os wich will require something like /c/myfile.exe instead of c:/myfile.exe.

Reply:
Can you add that detail to the docs, please?